static/js/tools/timeline_chart.tsx

```diff
@@ -197,6 +196,7 @@ class ChartRegion extends Component<ChartRegionPropsType, {}> {
         statsVarTitle[statsVar] = this.props.statsVars[statsVar].title;
       }
       plotParams.title = statsVarTitle;
+      plotParams.source = Array.from(statsData.data.sources);
```

Comment:
In my other PR, I am changing the plotParams to be keyed by label and value with only color and style.

The source is technically not part of a "plot". so can just pass the source as argument of drawGroupLineChart(), no need to change the PlotParams struct